main/zoe/api/zoeGoverningContractFacet.md

```diff
@@ -0,0 +1,157 @@
+# Zoe Governing Contract Facet
+
+### <span style="color:red">Need clarification: In many methods the instanceHandle is referenced in the chaimail file, but in the code it is not used. Affected methods: reallocate, complete, burnEscrowReceipt</span>
```

Comment:
Ah, good catch. It should be removed. Zoe creates a new contract facet per contract instance, and remembers the instanceHandle and fills it in as needed, which allowed us to remove it from the API